tests/Bridge/Exporter/AdminExporterTest.php

```diff
@@ -59,7 +59,7 @@ public function testGetExportFilename(): void
             ->method('getClass')
             ->willReturn('MyProject\AppBundle\Model\MyClass');
         $adminExporter = new AdminExporter(new Exporter());
-        $this->assertRegExp(
+        $this->assertMatchesRegularExpression(
```

Comment:
Looks like you can't install a version of PHPUnit that has this method on 7.2. Maybe do this?
```suggestion
        $method = method_exists($this, 'assertMatchesRegularExpression')? 'assertMatchesRegularExpression' : 'assertRegExp';
        $this->$method(
```

Reply:
I think I can fix it bumping phpunit-bridge: https://github.com/symfony/symfony/pull/37960

Reply:
Yes, fixed